Index: iphonedevonlinux/apps/HelloWorldFirst/Makefile
===================================================================
--- iphonedevonlinux/apps/HelloWorldFirst/Makefile	(revision 83)
+++ iphonedevonlinux/apps/HelloWorldFirst/Makefile	(working copy)
@@ -1,5 +1,4 @@
-#CC = /path/to/my/toolchain/bin/arm-apple-darwin9-gcc
-CC=arm-apple-darwin9-gcc
+CC = /home/debian/psycle/branches/bohan/iphone_cross_pkg/iphonedevonlinux/toolchain/pre/bin/arm-apple-darwin9-gcc
 LD = $(CC)
 
 # We build our first hello world.
@@ -17,6 +16,8 @@
 		-bind_at_load \
 		-w
 
+LDFLAGS = -bind_at_load -w
+
 all:	HelloWorldFirst
 
 HelloWorld1: HelloWorldFirst.o
Index: iphonedevonlinux/keys.list
===================================================================
--- iphonedevonlinux/keys.list	(revision 83)
+++ iphonedevonlinux/keys.list	(working copy)
@@ -12,3 +12,4 @@
 2.0	5A258F	198d6602ba2ad2d427adf7058045fff5f20d05846622c186cca3d423ad03b5bc3f43c61c
 2.0	5A240D	e24bfab40a2e5d3dc25e089291846e5615b640897ae8b424946c587bcf53b201a1041d36
 2.0	5A225C	ea14f3ec624c7fdbd52e108aa92d13b16f6b0b940c841f7bbc7792099dae45da928d13e7
+3.1.2	7D11	47D76295817F74953F8E557B4917FE2201E9778A9900E43FBF311A83F176FE521B996A4B
Index: iphonedevonlinux/firmware.list
===================================================================
--- iphonedevonlinux/firmware.list	(revision 83)
+++ iphonedevonlinux/firmware.list	(working copy)
@@ -15,4 +15,4 @@
 2.0.1	iPhone(2G)	http://appldnld.apple.com.edgesuite.net/content.info.apple.com/iPhone/061-5135.20080729.Vfgtr/iPhone1,1_2.0.1_5B108_Restore.ipsw
 2.0	iPhone(3G)	http://appldnld.apple.com.edgesuite.net/content.info.apple.com/iPhone/061-4955.20080710.bgt53/iPhone1,2_2.0_5A347_Restore.ipsw
 2.0	iPhone(2G)	http://appldnld.apple.com.edgesuite.net/content.info.apple.com/iPhone/061-4956.20080710.V50OI/iPhone1,1_2.0_5A347_Restore.ipsw
-
+3.1.2	iPhone(3GS)	http://appldnld.apple.com.edgesuite.net/content.info.apple.com/iPhone/061-7270.20091008.phn32/iPhone2,1_3.1.2_7D11_Restore.ipsw
Index: iphonedevonlinux/toolchain.sh
===================================================================
--- iphonedevonlinux/toolchain.sh	(revision 83)
+++ iphonedevonlinux/toolchain.sh	(working copy)
@@ -25,10 +25,11 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 
 # What version of the toolchain are we building?
-TOOLCHAIN_VERSION="3.1.2"
+#TOOLCHAIN_VERSION="3.1.2"
+TOOLCHAIN_VERSION="3.2"
 
 #what device are we building for?
-DEVICE="iPhone_3G"
+DEVICE="iPhone_3GS"
 
 # Everything is built relative to IPHONEDEV_DIR
 IPHONEDEV_DIR="`pwd`"
@@ -124,7 +125,7 @@
 MNT_DIR="${FILES_DIR}/mnt"
 FW_DIR="${FILES_DIR}/firmware"
 
-IPHONE_SDK="iphone_sdk_*.dmg"
+IPHONE_SDK="*iphone_sdk_*.dmg"
 [ -z $IPHONE_SDK_DMG ] && IPHONE_SDK_DMG="${FILES_DIR}/${IPHONE_SDK}"
 
 # URLS
